themes/default/content/docs/guides/crosswalk/aws/ecr.md

```diff
@@ -216,10 +216,9 @@ After building and tagging, we then need to authenticate with the ECR registry.
 single registry and lasts 12 hours. The AWS CLI provides an easy way to do this:
 
 ```bash
-$ $(aws ecr get-login --no-include-email)
+$ aws ecr get-login-password --region region | docker login --username AWS --password-stdin aws_account_id.dkr.ecr.region.amazonaws.com
```

Comment:
For consistency with the content before and after this line, lets keep the account id and region explicit.

```suggestion
$ aws ecr get-login-password | docker login --username AWS --password-stdin 012345678901.dkr.ecr.us-west-2.amazonaws.com
```